docs/guides/VTEX-Platform-Overview/store-architecture/store-architecture-use-cases.md

```diff
@@ -0,0 +1,163 @@
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2b.png)
+
+The B2B Suite is a set of VTEX IO apps tailored to streamline B2B store management. This collection of apps empower the store to oversee organizations (the companies enable to make purchases in the store), as well as other functions and permissions for storefront and checkout. For instance, it prevents an organization from changing its address, offers special price tables, custom payments methods, user roles, among others settings.
+
+The main account has more modules than the sellers, sharing just some, such as the [Order Management System (OMS)](https://developers.vtex.com/docs/guides/orders-overview) and [Catalog](https://developers.vtex.com/docs/guides/catalog-overview), as you can see in the diagram below.
+
+![image](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/docs/guides/VTEX-Platform-Overview/store-architecture/basic-b2b-suite.png)
+
+Declaring these modules in the architecture not only makes the process clearer but also makes the visualization of player responsibilities easier to understand. In this case, for example, [Master Data](https://developers.vtex.com/docs/guides/master-data-introduction) remains with the sellers because customer approval is done at the seller level.
+
+In B2B scenarios, real-time communication between the B2B Suite and sellers is common, as the information is always up-to-date, and also for any calculations of fees and taxes at checkout, since the end customer needs to have a smooth purchasing experience.
+
+In the diagram above you can see the synchronous communication between [Organizations](https://developers.vtex.com/docs/apps/vtex.b2b-organizations) component in the main account and the sellers.
+
+### Why choose this architecture?
+
+- **Streamlined Organization management**: Fosters efficient management of multiple organizations from a centralized platform by grouping storefront users into organizations and dividing them into cost centers. Shared order history, payment terms, and price lists enhance efficiency and promote cohesive collaboration across the organization.
+- **Customized user experience**: Tailor the user experience for each organization or user group, providing personalized pricing, payment methods, and product catalogs to meet specific business needs.
+- **Enhanced security and control**: Assign roles and permissions to users, ensuring appropriate access levels and maintaining control over sensitive information and transactions.
+- **Efficient Order Processing**: Simplify the checkout process with shared shipping addresses for cost centers, reducing eros and streamlining order fulfillment.
+- **Improved communication and collaboration**: Facilitate communication between users within an organization and streamline the creation of new organizations through request forms and approval queues.
```

Comment:
"Facilitate communication between users within an organization" - how?

Maybe explain that they all get to see and manage the orders within their organization (or cost center?) provided that they have the required roles

Reply:
In the part about creation of new organizations / request and approval queues, maybe link https://developers.vtex.com/docs/apps/vtex.b2b-organizations